schema/defs.json

```diff
@@ -61,6 +61,12 @@
         }
       ]
     },
+    "base64": {
+      "type": "string",
+      "media": {
+        "binaryEncoding": "base64"
```

Comment:
Shouldn't this be `contentEncoding`?
(http://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentencoding)

Reply:
Not in draft 4 which we are using. See https://json-schema.org/specification-links.html which links to https://datatracker.ietf.org/doc/html/draft-luff-json-hyper-schema-00#section-4.3.1.1

Reply:
Doh, sorry @vbatts :upside_down_face: :heart: